app/electron/preload/webview/elements/move/stub.ts

```diff
@@ -14,27 +28,55 @@ export function createStub(el: HTMLElement) {
     stub.style.backgroundColor = 'rgba(0, 0, 0, 0.2)';
     stub.style.display = 'none';
 
+    // Store the original position information
+    const parent = el.parentElement;
```

Comment:
The original stub position is already created and stored here. The stub should be free to move anywhere regardless of its original position. There's already resolution logic for when an element is placed in its original location as you can see in the drag functionalities here. Let me know if this helps. 

https://github.com/onlook-dev/onlook/blob/cdbebe99f07a154ef5ea5d8a89b3edecce3e844f/app/electron/preload/webview/elements/move/drag.ts#L21